components/servo/webview_delegate.rs

```diff
@@ -455,6 +515,8 @@ pub trait WebViewDelegate {
     /// Request to stop a haptic effect on a connected gamepad.
     fn stop_gamepad_haptic_effect(&self, _webview: WebView, _: usize, _: IpcSender<bool>) {}
 
+    fn show_form_control(&self, _webview: WebView, _form_control: FormControl) {}
```

Comment:
Sorry. This comment got lost somehow:

We were trying to keep all the "show"/"hide" methods together so that the rustdoc was easier to read. This didn't happen with `show_notification` unfortunately. Please move this under `hide_ime`.

```suggestion
    /// Request that the embedder show UI elements for form controls that are not integrated
    /// into page content, such as dropdowns for `<select>` elements.
    fn show_form_control(&self, _webview: WebView, _form_control: FormControl) {}
```